src/Tes.SDK/TesClient.cs

```diff
@@ -80,6 +80,10 @@ public TesClient(Uri baseUrl, string username, string password)
             ArgumentException.ThrowIfNullOrEmpty(password);
         }
 
+        public TesClient(string scheme, TesCredentials tesCredentials)
```

Comment:
seems like we should set scheme to "https" by default, or at the very least, have a /// so the developer knows what to put in there